manifest/2.1.2.xml

```diff
@@ -70,7 +70,7 @@
 
     <project name="gomemcached" path="godeps/src/github.com/couchbase/gomemcached" remote="couchbase" revision="0da75df145308b9a4e6704d762ca9d9b77752efc"/>
 
-    <project name="sg-bucket" path="godeps/src/github.com/couchbase/sg-bucket" remote="couchbase" revision="e774e379a0452bfff199a69e053dc565918d680c"/>
+    <project name="sg-bucket" path="godeps/src/github.com/couchbase/sg-bucket" remote="couchbase" revision="release/2.1.2.1"/>
```

Comment:
I only see a `feature/2.1.2_MP1` branch on sg-bucket. Are we awaiting a PR or something there?